YOUR_ADMIN_FOLDER/widget.php

```diff
@@ -0,0 +1,51 @@
+<?php
```

Comment:
The code here is just gotten from YOUR_ADMIN_FOLDER/tawk_to_widget.php. Transferred it here since there were problems encountered for ZenCart 1.5.7 for having the logic in the same file.

Reply:
`widget` is far too generic a name and has high chance of conflicts. Perhaps something like `tawk_widget_manager.php` to avoid chances of conflicts